TRIAGING.md

```diff
@@ -0,0 +1,151 @@
+
+There are a number of labels that have been defined for the different `conda` projects. In order to create a streamlined triaging process, label terminologies are standardized using similar (if not the same) labels.
+
+
+### How are new labels added?
+
+New **global** labels (_i.e._, generic labels that apply equally to all `conda` repos) can be added to the `conda/infra`'s [`.github/global.yml` file](https://github.com/conda/infra/blob/main/.github/global.yml); new **local** labels (_i.e._, labels specific to particular `conda` repos) can be added to each repository's `.github/labels.yml` file (_e.g._, `conda/conda`'s [label file](https://github.com/conda/conda/blob/master/.github/labels.yml)). All new labels should follow the labeling syntax described in the "How are new labels defined?" section of this document.
+
+
+### Are there any templates to use as responses for commonly-seen issues?
+
+Some of the same types of issues appear regularly (_e.g._, issues that are duplicates of others, tickets that should be filed in the Anaconda issue tracker, errors that are due to a user's specific setup/environment, etc.).
+
+Below are some boilerplate responses for the most commonly-seen triaging issues:
+
+##### Duplicate Issue
+
+"Closing this issue since this is a duplicate of **[link to primary issue]**; please feel free to add any additional discussion items to that issue directly."
+
+> **Note:** In addition to posting the above comment, make sure to apply the `duplicate` label on the issue being closed and `duplicate::primary` to the issue which was filed originally.
+
+##### Ticket Should be in the Anaconda Issue Tracker
+
+"Thank you for filing this issue! Due to the contents of this ticket being off-topic for this repo, we ask that you please file a new one in the [Anaconda issue tracker](https://github.com/ContinuumIO/anaconda-issues/issues) (it is the place for `conda` installer/package manager issues) if you are still encountering this problem."
+
+> **Note:** Be sure to apply the `off-topic` label for these tickets before closing them out.
+
+##### Requesting an Uninstall/Reinstall of `conda`
+
+"Please uninstall your current version of `conda` and reinstall the latest version. Feel free to use either the [miniconda](https://docs.conda.io/en/latest/miniconda.html) or [anaconda](https://www.anaconda.com/products/individual#windows) installer, whichever is more appropriate for your needs."
+
+##### "Out of Scope" Issue
+
+"Unfortunately, this issue is outside of the scope of support via GitHub; if you continue to experience the problems described here, please post details to the [Nucleus forums](https://community.anaconda.cloud/)."
+
+
+In order to not have to manually type or copy/paste the above repeatedly, please note that it's possible to add text for the most commonly-used responses via [GitHub's "Add Saved Reply" option](https://docs.github.com/en/get-started/writing-on-github/working-with-saved-replies/creating-a-saved-reply).
```

Comment:
Let's rework this into foldable `<details>` blocks to make it more concise. Let's also wrap the template text in `<pre>` tags to make them easier to copy/paste (while still preserving the ability to boldface important text). Something like this:

> ### Are there any templates to use as responses for commonly-seen issues?
> 
> Some of the same types of issues appear regularly (_e.g._, issues that are duplicates of others, tickets that should be filed in the Anaconda issue tracker, errors that are due to a user's specific setup/environment, etc.).
> 
> Below are some boilerplate responses for the most commonly-seen triaging issues. In order to not have to manually type or copy/paste the above repeatedly, please note that it's possible to add text for the most commonly-used responses via [GitHub's "Add Saved Reply" option](https://docs.github.com/en/get-started/writing-on-github/working-with-saved-replies/creating-a-saved-reply).
> 
> <details>
> <summary><b>Duplicate Issue</b></summary>
> 
> <pre>
> This is a duplicate of <b>[link to primary issue]</b>;
> feel free to continue the discussion there.
> </pre>
> 
> > :warning: Apply the https://github.com/conda/infra/labels/duplicate label to the issue being closed and https://github.com/conda/infra/labels/duplicate%3A%3Aprimary to the original issue.
> 
> </details>
>
> <details>
> <summary><b>Requesting an Uninstall/Reinstall of <code>conda</code></b></summary>
> 
> <pre>
> Please uninstall your current version of `conda` and reinstall the latest version.
> Feel free to use either the [miniconda](https://docs.conda.io/en/latest/miniconda.html)
> or [anaconda](https://www.anaconda.com/products/individual#windows) installer,
> whichever is more appropriate for your needs.
> </pre>
> 
> </details>
>
> <details>
> <summary><b>Redirect to Anaconda Issue Tracker</b></summary>
> 
> <pre>
> Thank you for filing this issue! Unfortunately, this is off-topic for this repo.
> If you are still encountering this issue please reopen in the
> [Anaconda issue tracker](https://github.com/ContinuumIO/anaconda-issues/issues)
> where `conda` installer/package issues are addressed.
> </pre>
> 
> > :warning: Apply the https://github.com/conda/infra/labels/off-topic label to these tickets before closing them out.
> 
> </details>
> 
> <details>
> <summary><b>Redirecting to Nucleus Forums</b></summary>
> 
> <pre>
> Unfortunately, this issue is outside the scope of support we offer via GitHub;
> if you continue to experience the problems described here,
> please post details to the [Nucleus forums](https://community.anaconda.cloud/).
> </pre>
> 
> > :warning: Apply the https://github.com/conda/infra/labels/off-topic label to these tickets before closing them out.
> 
> </details>